Makefile

```diff
@@ -41,27 +47,37 @@ test:
 	done
 .PHONY: test
 
-prep-for-run:
+copy-k8s-sa:
+	mkdir -p ${TLS_CERTS_BASEDIR} || true
+	oc -n ${NAMESPACE} get pod -l component=elasticsearch -o jsonpath={.items[0].metadata.name} > _output/espod && \
+	oc -n ${NAMESPACE} exec -c elasticsearch $$(cat _output/espod) -- cat /var/run/secrets/kubernetes.io/serviceaccount/ca.crt > _output/ca.crt && \
+	oc -n ${NAMESPACE} serviceaccounts get-token elasticsearch > _output/sa-token && \
+	echo ${NAMESPACE} > _output/namespace && \
+	sudo mkdir -p /var/run/secrets/kubernetes.io/serviceaccount/||:  && \
+	sudo ln -sf $${PWD}/_output/ca.crt /var/run/secrets/kubernetes.io/serviceaccount/ca.crt && \
+	sudo ln -sf $${PWD}/_output/sa-token /var/run/secrets/kubernetes.io/serviceaccount/token
```

Comment:
we really need  to fix this - we should never have to use sudo for local testing - this means we need a way to pass in the directory where the token/cert are, or perhaps individually specify the path to the ca.crt and token

Reply:
See the previous comment. Please open a separate issue for this.

Reply:
I believe `/var/run/secrets/kubernetes.io/serviceaccount` is an unconfigurable path used by the `client-go`. It may be with @pavolloffay changes to how it sources the kubeconfig that this is no longer needed to run locally.  If the `hacking.md` is correct I believe @pavolloffay could remove this.

Reply:
This might be useful if you want to use exactly the same service account as deployed elasticsearch proxy is using as the locally used k8s account might be different.

I can remove these in a separate PR.

Reply:
Added via #6